releases/2021-12-24-releasenotes.md

```diff
@@ -0,0 +1,17 @@
+---
+title: 'Release Notes: Smart Contracts Improvements'
```

Comment:
```suggestion
title: 'Improving Smart Contracts Validation'
```

Reply:
I don't really care where we land on title...I would not start a post with `Release Notes: ` since it's implied.

Reply:
I chatted with @ryansb about this one. Even though it's implied in the URL it may not necessarily be clear for things like RSS feeds. Having `Release Notes:` in the title makes it clear to all parties.

Reply:
It's more conventional to use `Release Notes` as a page heading vs. prefixing to each and every post?  Adding `Release Notes` to the title of each and every post seems odd and unnecessary.

https://changelog.kolide.com/
https://releasenotes.carta.com/
https://docs.aws.amazon.com/lambda/latest/dg/lambda-releases.html